app/2.0/start/toolbox/deploy.md

```diff
@@ -170,7 +181,7 @@ guide](https://www.firebase.com/docs/hosting/quickstart.html).
         "rules": "database.rules.json"
       },
       "hosting": {
-        "public": "build/default",
+        "public": "build/es5-bundled",
```

Comment:
Minor: Update to `"build/es5-bundled/"`.
In the PSK (and here in the "Build for deployment" we always use `/` to 'close' a folder name.